docs/src/lib/representations.md

```diff
@@ -184,6 +184,14 @@ tohrep(::VPolygon{Float64})
 tovrep(::VPolygon{Float64})
 ```
 
+### Sorting directions
+
+```@docs
+jump2pi
+<=(::AbstractVector{Float64}, ::AbstractVector{Float64})
```

Comment:
Do we also want to document the rational version?

Reply:
actually it gets documented with this line (don't ask why).

Reply:
okay... :)